src/hotspot/share/prims/jvm.cpp

```diff
@@ -496,6 +499,15 @@ JVM_LEAF(jboolean, JVM_IsUseContainerSupport(void))
   return JNI_FALSE;
 JVM_END
 
+JVM_LEAF(jboolean, JVM_IsContainerized(void))
+#ifdef LINUX
+  if (OSContainer::is_containerized()) {
```

Comment:
// nit: personal preference...

return OSContainer::isContainerized() ? JNI_TRUE : JNI_FALSE;

Reply:
I've kept this as is, since the suggestion generates this code after preprocessing on Linux:

```
return OSContainer::is_containerized() ? JNI_TRUE : JNI_FALSE;
return JNI_FALSE;
```

over the existing version:

```
if (OSContainer::is_containerized()) {
    return JNI_TRUE;
}
return JNI_FALSE;
```